src/file-2.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:44933/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:44933/project/issues?id=sample&pullRequest=5593&issues=AWcr4XT0SmAvUu5amEqV&open=AWcr4XT0SmAvUu5amEqV)

[ISSUE_KEY]: AWcr4XT0SmAvUu5amEqV